version_check.py

```diff
@@ -0,0 +1,28 @@
+"""Script for checking the correct versions of Python, StrawberryFields and TensorFlow are being
+used."""
+import sys
+
+import strawberryfields as sf
+import tensorflow as tf
+
+python_version = sys.version_info
+sf_version = sf.__version__
+tf_version = tf.__version__.split(".")
+
+if python_version > (3, 6):
+    raise SystemError("Your version of python is {}.{}. You must have Python 3.5 or 3.6 installed "
+                      "to run this script.".format(python_version.major, python_version.minor))
+
+if python_version < (3, 5):
+    raise SystemError("Your version of python is {}.{}. You must have Python 3.5 or 3.6 installed "
+                      "to run this script.".format(python_version.major, python_version.minor))
```

Comment:
Could these be merged into a single check? `if python_version < (3, 5) or python_version > (3, 6)`.

Wait, is comparison between tuples distributive in Python? I didn't know that!

Reply:
Oh good spot, have updated!